<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WkfLogInteractiveWidget</class>
 <widget class="QWidget" name="WkfLogInteractiveWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>852</width>
    <height>426</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="optionsButton">
         <property name="text">
          <string>选项</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="errorCount">
         <property name="text">
          <string>0 错误</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="warningCount">
         <property name="text">
          <string>0 警告</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="expandButton">
         <property name="minimumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="toolTip">
          <string>全部展开</string>
         </property>
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="collapseButton">
         <property name="minimumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="toolTip">
          <string>全部折叠</string>
         </property>
         <property name="text">
          <string>-</string>
         </property>
         <property name="arrowType">
          <enum>Qt::NoArrow</enum>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLineEdit" name="searchBar">
         <property name="placeholderText">
          <string>查找...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QWidget" name="optionsWidget" native="true">
       <layout class="QVBoxLayout" name="optionsLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="filterLabel">
            <property name="text">
             <string>包含:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="fatalCheckBox">
            <property name="text">
             <string>致命错误</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="errorCheckBox">
            <property name="text">
             <string>错误</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="warningCheckBox">
            <property name="text">
             <string>警告</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="infoCheckBox">
            <property name="text">
             <string>信息</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="debugCheckBox">
            <property name="text">
             <string>调试</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="developerCheckBox">
            <property name="text">
             <string>开发者</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="sortLabel">
            <property name="text">
             <string>排序方式:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="chronologicalButton">
            <property name="text">
             <string>按时间</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="typeButton">
            <property name="text">
             <string>类型</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="alphabeticalButton">
            <property name="text">
             <string>按字母顺序</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="reverseCheckBox">
            <property name="text">
             <string>反向</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="wkf::log::TreeView" name="treeView">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ContiguousSelection</enum>
     </property>
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>wkf::log::TreeView</class>
   <extends>QTreeView</extends>
   <header>logging/WkfLogTreeView.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
